site/en/docs/extensions/mv3/known-issues/index.md

```diff
@@ -57,11 +49,21 @@ This section lists major features that will be added to the Manifest V3 platform
 
 **Tracking issue:** [1247690][]
 
-### Service workers are not started in response to webRequest events
+## Fixed but not yet in Stable {: #resolved }
```

Comment:
This page is already pretty heavy with headings. Here's my suggestion. Move `onAuthRequired` to the top of the Capabilities section and replace the estimted timeline with "**Shipped:** 108".

Reply:
Ack.